packages/react-components/src/components/CameraButton.tsx

```diff
@@ -248,6 +250,10 @@ export const CameraButton = (props: CameraButtonProps): JSX.Element => {
     }
   };
 
+  const splitButtonMenuProps: IButtonProps = {
```

Comment:
avoid defining consts on the render method, pull this outside the render body (otherwise use useMemo)

Reply:
+1

Reply:
done